02.calendar/calendar.js

```diff
@@ -0,0 +1,42 @@
+const argv = require('minimist')(process.argv.slice(2))
+const date = new Date()
+
+// 今年もしくは引数の年
+let year = date.getFullYear()
+if (argv.y != null) {
+  year = argv.y
+}
+
+// 今月もしくは引数の月
+let month = date.getMonth() + 1
+if (argv.m != null) {
+  month = argv.m
+}
+
+// 月初
+const firstDay = 1
+// 月初の曜日
+let firstWday = new Date(year, month - 1, 1).getDay()
+// 月末
+const lastDay = new Date(year, month, 0)
+const lastDayNum = lastDay.getDate()
+
+const title = `      ${year}` + '年' + `${month}` + '月'
```

Comment:
せっかくtemplate literal使えているので、 `+` も排除してしまってはどうでしょうか
```suggestion
const title = `      ${year}年${month}月`
```